PARAMETERS_MAPPING/unit_view.csv

```diff
@@ -0,0 +1,61 @@
+id;name;short_name;definition;date_added
+451;Bytes;Bytes                                                                                                         ;;2013-07-18
+769;Cells per millilitre;Cells ml-1                                                                                                    ;;2014-07-10
+445;Decibars;dBar                                                                                                          ;;2013-07-18
+439;Decibels;dbel                                                                                                          ;;2013-07-18
```

Comment:
Decibars should be abbreviated "dbar"
and the abbreviation for decibel is "dB"

unit symbol are lower case unless the unit is derived from a proper name

Reply:
Thanks @bpasquer, those have been fixed in https://github.com/aodn/data-services/pull/793/commits/1f1d46e36ffe556b6d3c21f8f4af576acfa046fc